app/data/compose/docker-compose-laconic-dot-com.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.2"
+
+services:
+  lasso:
+    image: cerc/laconic-dot-com:local
+    restart: always
+    ports:
+      - "0.0.0.0:3000:3000"
```

Comment:
If deployed via the "deployment" mechanism, I think this can be `- "3000"`

Reply:
yep, instructions added to the readme